src/main/java/duke/AddDeadLineCommand.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Objects;
+
+public class AddDeadLineCommand extends Command {
+
+    AddDeadLineCommand(String input) {
+        this.input = input;
+    }
+
+    /**
+     * Processes input given by the user to build a deadLine Task
+     *
+     * @param input
+     */
+    Task process(String input) throws DukeException, ParseException {
+        ArrayList<String> commands = new ArrayList<>();
+        commands.add("deadline");
+        commands.add("/by");
+
+        String[] deadLineItems = Arrays.stream(Parser.ParseString(input, commands)).map(String::trim).toArray(String[]::new);
```

Comment:
I think you can stack up these functions in one line so that it doesn't exceed 120 characters:
```suggestion
        String[] deadLineItems = Arrays.stream(Parser.ParseString(input, commands))
            .map(String::trim)
            .toArray(String[]::new);
```